*** wrapper-gtk24.sh.orig	Wed May 18 14:23:43 2005
--- wrapper-gtk24.sh	Wed May 18 17:06:10 2005
***************
*** 84,90 ****
  
     # Append any libraries that are still missing.
     export LD_LIBRARY_PATH="`LANGUAGE=C LANG=C ldd "$binary" | vm_append_missing`"
- 
     #
     # "$binary"'s stdout goes unmodified to the shell's stdout.
     # "$binary"'s stderr goes almost unmodified to the shell's stderr
--- 84,89 ----
***************
*** 167,173 ****
  
     vm_run "$@"
     exitCode="$?"
!    if [ "$exitCode" -eq 0 ]; then
        # Assume that the system's environment works. Then we are done.
        exit "$exitCode"
     fi
--- 166,172 ----
  
     vm_run "$@"
     exitCode="$?"
!    if [ "$exitCode" -eq -1 ]; then
        # Assume that the system's environment works. Then we are done.
        exit "$exitCode"
     fi
***************
*** 181,186 ****
--- 180,191 ----
     VMWARE_USE_SHIPPED_GTK=yes
  fi
  
+ # Local mod -- bmc
+ _save_LD_LIBRARY_PATH=$LD_LIBRARY_PATH
+ LD_LIBRARY_PATH=
+ VMWARE_USE_SHIPPED_GTK=yes
+ # End local mod
+ 
  if [ "$VMWARE_USE_SHIPPED_GTK" = 'yes' ]; then
     #
     # Libraries
***************
*** 223,227 ****
--- 228,273 ----
     fi
  fi
  
+ # Local mod -- bmc
+ #vm_append_lib libXft.so.2
+ #vm_append_lib libXrender.so.1
+ #vm_append_lib libart_lgpl_2.so.2
+ #vm_append_lib libatk-1.0.so.0
+ #vm_append_lib libatkmm-1.6.so.1
+ vm_append_lib libcrypto.so.0.9.7
+ #vm_append_lib libexpat.so.0
+ #vm_append_lib libfontconfig.so.1
+ #vm_append_lib libfreetype.so.6
+ #vm_append_lib libgcc_s.so.1
+ #vm_append_lib libgdk-x11-2.0.so.0
+ #vm_append_lib libgdk_pixbuf-2.0.so.0
+ #vm_append_lib libgdkmm-2.4.so.1
+ #vm_append_lib libglade-2.0.so.0
+ #vm_append_lib libglib-2.0.so.0
+ #vm_append_lib libglibmm-2.4.so.1
+ #vm_append_lib libgmodule-2.0.so.0
+ #vm_append_lib libgnomecanvas-2.so.0
+ #vm_append_lib libgnomecanvasmm-2.6.so.1
+ #vm_append_lib libgobject-2.0.so.0
+ #vm_append_lib libgtk-x11-2.0.so.0
+ #vm_append_lib libgtkmm-2.4.so.1
+ #vm_append_lib libpango-1.0.so.0
+ #vm_append_lib libpangoft2-1.0.so.0
+ #vm_append_lib libpangomm-1.4.so.1
+ #vm_append_lib libpangox-1.0.so.0
+ #vm_append_lib libpangoxft-1.0.so.0
+ #vm_append_lib libpixops.so.2.0.1
+ #vm_append_lib libpng12.so.0
+ #vm_append_lib librsvg-2.so.2
+ #vm_append_lib libsigc-2.0.so.0
+ vm_append_lib libssl.so.0.9.7
+ #vm_append_lib libstdc++.so.5
+ #vm_append_lib libxml2.so.2
+ 
+ LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${_save_LD_LIBRARY_PATH}
+ export LD_LIBRARY_PATH
+ 
+ # End local mod
+ 
  vm_run "$@"
  exit 0
